.github/CODEOWNERS

```diff
@@ -40,6 +40,9 @@
 
 /.github/CODEOWNERS @odoo/rd-code-owners
 
+/addons/account*/ @odoo/rd-accounting
+/addons/l10n_*/ @odoo/rd-accounting
```

Comment:
Should be moved up in the `# Generic fallback rules` section due to the use of `*` in the module part. (just before website)